OpenUpSA/wazimap-ng
# Feature/cache profile points download

## Description

Caching for the download points URLs.

## How to test it locally

Set `DEBUG_CACHE_AS_PRODUCTION` to `True` and then `curl -I "http://localhost:8000/api/v1/profile/1/points/geography/ZA/points/?format=json"` and you should see the header:
```
Cache-Control: max-age=3600
```

The first time you do it it is slow as there is no cache. Then the next time, within one hour, will be near instant.
